src/Solnet.Rpc/Builders/TransactionInstructionFactory.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using Solnet.Wallet;
+using Solnet.Rpc.Models;
+
+namespace Solnet.Rpc.Builders
+{
+    /// <summary>
+    /// Methods to create instances of the TransactionInstruction for languages that do not support init-only setters
+    /// </summary>
+    public static class TransactionInstructionFactory
+    {
+
+        /// <summary>
+        /// Helper method to make TransactionInstruction objects available to outside Program implementations in 
+        /// languages that do not currently support immutable init-only setters like VB.Net
+        /// </summary>
+        /// <param name="programId"></param>
+        /// <param name="keys"></param>
```

Comment:
Would just like to see these comments here, looks good to me otherwise :)

Reply:
Sorted ✔️